scripts/speaker_tasks/create_librispeech_alignment_manifest.py

```diff
@@ -0,0 +1,121 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import os
+import random
+
+from nemo.collections.asr.parts.utils.manifest_utils import read_manifest, write_manifest
+
+random.seed(42)
+
+"""
+This script creates a manifest file containing word alignments.
+
+The alignments are obtained from: https://github.com/CorentinJ/librispeech-alignments
+"""
+
+# get librispeech examples without alignments for the desired dataset
+def get_unaligned_examples(unaligned_path, dataset):
+    with open(unaligned_path, 'r', encoding='utf-8') as f:
+        lines = f.readlines()
+        i = 0
+        skip_files = []
+        while i < len(lines):
+            l = lines[i].strip('\n')
+            if l[0] == '#':
+                unaligned_dataset = l.split(" ")[1]
+            elif unaligned_dataset == dataset:
+                unaligned_file = l.split(" ")[0]
+                skip_files.append(unaligned_file)
+            i += 1
+    return skip_files
+
+
+def main():
+    input_manifest_filepath = args.input_manifest_filepath
+    base_alignment_path = args.base_alignment_path
+    output_path = args.output_path
+    dataset = args.dataset
+
+    manifest = read_manifest(input_manifest_filepath)
+    target_manifest = []
+    unaligned_path = os.path.join(base_alignment_path, "unaligned.txt")
```

Comment:
this is a bit confusing if base_alignment_path should be path to `LibriSpeech` or `LibriSpeech/..` 
After looking at the code I understood it should be `PWD` of the `LibriSpeech/..` not `LibriSpeech` itself. Can you make it clearer what this base alignment path is? Actually, I felt unaligned.txt should be part of the LibriSpeech folder, but it's outside of it